instrumentation/servlet/servlet-3.0/javaagent-unit-tests/src/test/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/snippet/InjectionTest.java

```diff
@@ -0,0 +1,100 @@
+package io.opentelemetry.javaagent.instrumentation.servlet.v3_0.snippet;
+
+import static io.opentelemetry.javaagent.instrumentation.servlet.v3_0.snippet.Injection.stringInjection;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import io.opentelemetry.javaagent.bootstrap.servlet.SnippetHolder;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import org.junit.jupiter.api.Test;
+
+public class InjectionTest {
+  public String readFile(String resourceName) throws IOException {
```

Comment:
and move to the bottom of the file (see https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#ordering-of-class-contents)
```suggestion
  private static String readFile(String resourceName) throws IOException {
```